Introduction/Introduction/Introduction/task.md

```diff
@@ -20,34 +20,34 @@ Embark on a coding journey with refactoring and design patterns – your path to
 
 <div class="hint" title="Course Plan"> 
 
-- **Section 1: Refactoring and its purpose**
-    - Lesson 1: What is code refactoring?
-    - Lesson 2: Common code issues and their impact on code quality
-    - Lesson 3: Refactoring techniques
-    - Lesson 4: Getting familiar with IDE refactoring features
-- **Section 2: Code style and Formatting**
-    - Lesson 1: What is code style and formatting?
+- **[Section 1: Refactoring and its purpose](course://RefactoringAndItsPurpose)**
+    - [Lesson 1: What is code refactoring?](course://RefactoringAndItsPurpose/WhatIsCodeRefactoring)
+    - [Lesson 2: Common code issues and their impact on code quality](course://RefactoringAndItsPurpose/CommonCodeIssuesAndTheirImpactOnCodeQuality)
+    - [Lesson 3: Refactoring techniques](course://RefactoringAndItsPurpose/RefactoringTechniques)
+    - [Lesson 4: Getting familiar with IDE refactoring features](course://RefactoringAndItsPurpose/GettingFamiliarWithIDERefactoringFeatures)
+- **[Section 2: Code style and Formatting](course://CodeStyleAndFormatting)**
+    - [Lesson 1: What is code style and formatting?](course://CodeStyleAndFormatting/WhatIsFormatting)
     - Lesson 2: Reformat the code using IDE
```

Comment:
It seems it's not a lesson anymore and it this line should be dropped, right?

Reply:
You're right, thanks! I will update this task